packages/workbox-cli/src/app.js

```diff
@@ -33,12 +33,23 @@ module.exports = async (command, configFile) => {
       break;
     }
 
+    case 'copyLibraries': {
+      assert(options, errors['missing-dest-dir-param']);
+      const parentDirectory = path.resolve(process.cwd(), options);
+
+      const dirName = await workboxBuild.copyWorkboxLibraries(parentDirectory);
```

Comment:
This is a bit of a weird one, but would it make sense for `copyWorkboxLibraries` to return the full path? That seems more useful than the dirname especially given `path.dirname(...)` would provide this.